### ISSUE TITLE
When using keyboard navigation, focus gets stuck in the edit message title field

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#28]
# Describe the bug
When navigating the page via keyboard only navigation, once the keyboard focus is placed on the "edit title" inline form element, it cannot be moved away unless the form is opened, and then the `cancel` button is activated.

## Steps To Reproduce
1. Go to a message display page `/messages/SOME_MESSAGE_ID` on a browser that supports keyboard navigation
2. Tab through until the message title is highlighted 
3. Attempt to tab away (either by pressing tab more or shift+tab to go back in the tab order)
4. See that the focus never leaves the message title

## Expected behavior
A person using keyboard navigation should be able to tab through all the elements without getting stuck.

**Comment by robby1066** [comment]
Looks like this is due to an event handler in the javascript that handles editing the message title. It was (seemingly) erroneously not letting key events fall through to their default behavior.

**Comment by robby1066** [comment]
This is related to #1

**Comment by robby1066** [comment]
Example of updating a message title using just keyboard navigation.


https://user-images.githubusercontent.com/342510/107127529-4bb02300-687c-11eb-9b95-7a945431aa47.mp4
